sdk/storage/azure-storage-queue/src/test/java/com/azure/storage/queue/spock/QueueAPITests.groovy

```diff
@@ -0,0 +1,485 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.queue.spock
+
+
+import com.azure.storage.queue.models.AccessPolicy
+import com.azure.storage.queue.models.SignedIdentifier
+import com.azure.storage.queue.models.StorageErrorCode
+import com.azure.storage.queue.models.StorageErrorException
+import org.junit.Assert
+import org.junit.Ignore
+import reactor.test.StepVerifier
+import spock.lang.Unroll
+
+import java.time.Duration
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertFalse
+import static org.junit.Assert.assertNull
+
+class QueueAPITests extends APISpec {
+    def queueClient
+
+    static def testMetadata = Collections.singletonMap("metadata", "value")
+    static def createMetadata = Collections.singletonMap("metadata1", "value")
+
+    def setup() {
+        primaryQueueServiceClient = queueServiceBuilderHelper(interceptorManager).buildClient()
+        queueClient = primaryQueueServiceClient.getQueueClient(testResourceName.randomName("queue", 16))
+    }
+
+    def "Create queue with shared key from queue client"() {
+        when:
+        def createQueueResponse = queueClient.create()
+
+        then:
+        QueueTestHelper.assertResponseStatusCode(createQueueResponse, 201)
+    }
+
+    // TODO: Will implement the test after introduce the sas token generator
+    @Ignore
```

Comment:
Switch this to Spock's ignore instead of JUnit's. (`import spock.lang.Ignore`)

Reply:
Done.